src/validate.rs

```diff
@@ -868,20 +868,25 @@ fn validate_repos(data: &Data, errors: &mut Vec<String>) {
         }
         for team_name in repo.access.teams.keys() {
             if !github_teams.contains(&(repo.org.clone(), team_name.clone())) {
+                let error_reason = if data.is_team_archived(team_name, &repo.org) {
+                    "is archived. Please remove the team access from the `[access.teams]` section of the repo"
```

Comment:
I'm wondering if something like this wouldn't be even easier to understand. Might require a change to the data model, though, to deserialize a repository without the `access.teams` field set. 🤔 

```suggestion
                    "is archived. Please remove the `[access.teams]` section in the repo's toml"
```

Reply:
this change sounds more like https://github.com/rust-lang/team/pull/1754
Why should we suggest removing the entire `[access.teams]` section of the repo here?
That particular team is archived, but not the repo.

Reply:
Oh yeah, my bad. When looking how `[access.teams]` was used in the repository TOMLs, I forgot that this was about an archived team. 🙈